### ISSUE TITLE
Stash editor plugin appears capable of executing arbitrary JS

### ISSUE THREAD

**Issue by sjothi** [mohamicorp/editor-for-stash-issues#35]
Hi Folks,
I was trying to test part of markdown syntax today and just for fun i did a copy/paste of Javascript code in the README.md and i clicked "Preview". Its actually executing the Javascript content.

Steps to reproduce:
1. Goto any README.md file in any repo.
2. Clear all its content and paste the below content in the README.md file,
<html> TEST </html>

<script type="text/javascript">
alert("THIS IS TEST");
</script>
1. Click "Preview"
   You will actually get an alert.
   I am hoping this is an issue.
   Let me know if you need more info.
   Stash Version: 3.11.4
   Plugin Version: 1.5.3

**Comment by mdavoodi** [comment]
Hey Sjothi, that's pretty weird. I don't see the content you pasted. Did you accidentally remove it?

**Comment by sjothi** [comment]
Sorry. I have missed it.

```
<html>
<script type="text/javascript">
alert("TEST");
</script>
</html>
```

**Comment by sjothi** [comment]
Any update here ?

**Comment by mdavoodi** [comment]
Verified the bug, currently working on a fix.

**Comment by mdavoodi** [comment]
The fix for this is now live for the most recent version of Stash and Bitbucket.

Here's a [link](https://marketplace.atlassian.com/plugins/com.mohamicorp.stash.plugin.editor-for-stash/versions).

**Comment by sjothi** [comment]
Thanks for quickly fixing the issue. 
So 1.5.4 and 2.1.1 are the fix versions ?

**Comment by mdavoodi** [comment]
Yes. 1.5.4 is the Stash version with the fix, and 2.1.1 has a new editor feature as well as the fix.
